Makefile

```diff
@@ -40,10 +40,11 @@ lint:
 dev: lint test
 
 black:
-	black setup.py samcli tests
+    # command "black <directory>" will skip some files, it is a bug, see https://github.com/psf/black/issues/1820
+	find setup.py samcli tests -type f -name "*.py" | xargs black
```

Comment:
Should we use `black --include build setup.py samcli tests` so that we don't deviate too much in the meantime?

Reply:
good point! I did a big more digging and found we have a configuration file 🙂